Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
It's more of a convenience to have it there, so we don't have to require it only when we want it. But I see the value of requiring it when we want it so I'll remove it from the gem and see how it goes! 

I know I could put the require at the top of the spec_helper file but at the same time I've seem numero project that juste includes `pry` when they want to use it. I'll use that opportunity to give it a try.